### PR TITLE
Make tests pass

### PR DESCRIPTION
**Pull request by iNecas**
https://github.com/theforeman/foreman_remote_execution/pull/5 + more rubocop fixes

### CODE REVIEW

**Comment by iNecas** [comment]
Running tests http://ci.theforeman.org/view/Plugins/job/test_plugin_pull_request/983/

**Comment by iNecas** [comment]
@ares Ok with this rubocop fixes: the foreman core has this policy turned off, but it's in the todo yml

File: lib/foreman_remote_execution/engine.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ class Engine < ::Rails::Engine
     initializer 'foreman_remote_execution.configure_assets', :group => :assets do
       SETTINGS[:foreman_remote_execution] =
         {:assets => {:precompile => [
-          'template_input.js',
```
Comment (ares) [inline lib/foreman_remote_execution/engine.rb]: I don't like this cop. Why it's harmful to keep comma after last item? I like to use it.

Comment (iNecas) [inline lib/foreman_remote_execution/engine.rb]: As you wish, turned that one off

Comment (ares) [inline lib/foreman_remote_execution/engine.rb]: thank you!

**Comment by iNecas** [comment]
@ares ACK?

**Comment by ares** [comment]
:+1:

**Comment by stbenjam** [comment]
Yay :raised_hands:

**Pull request merged**